tests/acceptance/features/webUIFiles/search.feature

```diff
@@ -153,3 +153,20 @@ Feature: Search
     And the user searches for "favorite" using the webUI
     Then folder "favorite folder" should be listed on the webUI
     And folder "not favorite folder" should be listed on the webUI
+
+  @issue-3044
+  Scenario: Delete file from search list
+    Given the following files have been deleted by user "user1"
+      | name          |
+      | lorem-big.txt |
+    And user "user1" has uploaded file with content "hello" to "lorem-big.txt"
+    And the user has reloaded the current page of the webUI
+    When the user searches for "lorem-big" using the webUI
+    Then file "lorem-big.txt" should be listed on the webUI
```

Comment:
we are not testing the search itself here, that is tested in other scenarios, we are testing the deletion of a file that is on the search-list.
So line 164 should be a Given step and we don't need the check in line 165

Reply:
Okay. Got it